code/modules/reagents/reagents/drugs.dm

```diff
@@ -273,3 +273,83 @@
 	M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
 	M.stats.addTempStat(STAT_COG, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
 	M.stats.addTempStat(STAT_ROB, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
+
+/datum/reagent/drug/roachbeer
+	name = "Kakerlakenbier"
+	id = "roachbeer"
+	description = "A green-ish substance made out of diplopterum, beer and fuel mixed with water. Doesn't look nor smell like beer..."
+	taste_description = "heady roach guts"
+	reagent_state = LIQUID
+	color = "#123a15"
+	metabolism = REM
+	nerve_system_accumulations = 40
+	addiction_chance = 60
+	overdose = REAGENTS_OVERDOSE
+	addiction_threshold = 15
+	sanity_gain_ingest = 2
```

Comment:
Double sanity increase for vagabonds would be too strong, making it almost as strong as meth.

Reply:
That's intended, and I know about the methe; I made it so the withdrawal and the addiction would be a real pain, so it's a breakdown sooner or later, unless you drink 'the beer' again.
Also, it will be hard to make "clean" beer, bc roaches have not only diplo, but also blattedin, protein, and other chems. Unless you have a ChemMaster of sorts, you have to somehow get rid of all that stuff; lest thou wish to get addicted to 69 substances and get 60 toxin damage.